applications/FluidDynamicsApplication/custom_conditions/stokes_wall_condition.cpp

```diff
@@ -144,7 +209,7 @@ void StokesWallCondition<TDim,TNumNodes>::ApplyNeumannCondition(MatrixType &rLoc
     double A = norm_2(Normal);
     Normal /= A;
 
-    // CAUTION: "Jacobian" is 2.0*A for triangles but 0.5*A for lines 
+    // CAUTION: "Jacobian" is 2.0*A for triangles but 0.5*A for lines
```

Comment:
I have a doubt here. Is Jacobian 0.25 * A for quad?

Reply:
Now using the method from geometry to compute the Jacobian determinant, so this is doubt is resolved.